### PR TITLE
fix the typo of error 

### PR DESCRIPTION
**Pull request by jianglingxia**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

**Review by luxas — COMMENTED** [review]
LGTM, thanks!

**Comment by luxas** [comment]
/lgtm

@k8s-bot ok to test

**Comment by luxas** [comment]
> I0529 16:39:49.721] cmd/kubeadm/app/cmd/init.go:208: undefined: err
I0529 16:39:49.722] cmd/kubeadm/app/cmd/init.go:209: undefined: err
I0529 16:39:49.722] cmd/kubeadm/app/cmd/init.go:210: undefined: err

@jianglingxia Please fix the tests

File: cmd/kubeadm/app/cmd/init.go [hunk 1]
```diff
@@ -158,13 +158,12 @@ func NewInit(cfgPath string, cfg *kubeadmapi.MasterConfiguration, skipPreFlight,
...
 	// Set defaults dynamically that the API group defaulting can't (by fetching information from the internet, looking up network interfaces, etc.)
-	err := setInitDynamicDefaults(cfg)
-	if err != nil {
+	if err := setInitDynamicDefaults(cfg); err != nil {
 		return nil, err
 	}
 
 	if !skipPreFlight {
-		fmt.Println("[preflight] Running pre-flight checks")
+		fmt.Println("[preflight] Running pre-flight checks.")
```
Comment (luxas) [inline cmd/kubeadm/app/cmd/init.go]: not necessary

File: cmd/kubeadm/app/cmd/init.go [hunk 2]
```diff
@@ -179,7 +178,7 @@ func NewInit(cfgPath string, cfg *kubeadmapi.MasterConfiguration, skipPreFlight,
 		// Try to start the kubelet service in case it's inactive
 		preflight.TryStartKubelet()
 	} else {
-		fmt.Println("[preflight] Skipping pre-flight checks")
+		fmt.Println("[preflight] Skipping pre-flight checks.")
```
Comment (luxas) [inline cmd/kubeadm/app/cmd/init.go]: not necessary

**Comment by jianglingxia** [comment]
I rebase the branch and found some content in the pr has merge ,so i close it ,thanks!

**Pull request closed without merging**